bitnami/minio/templates/_helpers.tpl

```diff
@@ -85,6 +85,36 @@ Get the credentials secret.
 {{- end -}}
 {{- end -}}
 
+{{/*
+Get the root user key.
+*/}}
+{{- define "minio.rootUserKey" -}}
+{{- if .Values.auth.existingSecret -}}
+    {{- if .Values.auth.rootUserSecretKey -}}
+        {{- printf "%s" (tpl .Values.auth.rootUserSecretKey $) -}}
+    {{- else -}}
+        {{- "root-user" -}}
+    {{- end -}}
+{{- else -}}
+    {{- "root-user" -}}
+{{- end -}}
+{{- end -}}
```

Comment:
We can simplify this method, what do you think?

```suggestion
{{- define "minio.rootUserKey" -}}
{{- if and .Values.auth.existingSecret .Values.auth.rootUserSecretKey -}}
    {{- printf "%s" (tpl .Values.auth.rootUserSecretKey $) -}}
{{- else -}}
    {{- "root-user" -}}
{{- end -}}
```

Reply:
Thanks, you're completely right